### DISCUSSION TITLE
bricked Sonoff 4CH

### DISCUSSION THREAD

**Discussion by jabonifacio** [arendst/Tasmota#20127] [Q&A]
I guys,

I'm here to get some confirmation.
So... I had a sonoff 4Ch flashed with tasmota maybe 3 or more years ago. on the last 12 month's it was not in use.
Yesterday i pluged in and voila... everything was working.
But then i made the stupid mistake of trying to update to the latest version via OTA....  

Now the device is bricked... no lights, no relay functionality, not connecting to wifi.
I've tried all the tricks in the book i knew, press 4 or 6 times, fast on/off x7 times, then i try to connect to by serial.... tried to flash tasmota with tasmotizer... nothing, tried to erase the flash with esptool...
esptool.py erase_flash
esptool.py v4.6.2
Found 1 serial ports
Serial port /dev/ttyUSB0
Connecting......................................

it just says conecting.....

Did i kill it? or is there some other way to brute force new firmware?

ps. I have no idea to what firmware it had

Thanks

**Comment by sfromis** [comment]
While you indeed may have soft-bricked it by not following the [upgrade flow](https://tasmota.github.io/docs/Upgrading/#upgrade-flow), that should not hinder you from reflashing it from scratch, where you need to put the device in flashing mode. On [your device](https://templates.blakadder.com/sonoff_4CH.html) (?) this should be by pushing button 1 before/while applying power from the USB/serial adapter.

**Reply by jabonifacio** [reply]
hi,

Yes...
When i said that i tried to re-flash with the serial, it was by putting it on flashing mode.

Thanks

**Reply by sfromis** [reply]
What you posted says that your attempt of putting it in flashing mode failed.

Or your wiring/cabling/power situation was bad.

**Reply by jabonifacio** [reply]
My wiring,power,adapter etc is ok. I know because when i was trying to reflash the bricked device i had another sonoff 4ch with stock firmware that i had to flash tasmota, and the flash worked without any problems..

So it might be as you said problems entering the flash mode...

Any idea??

Thanks
Bonifácio

**Reply by sfromis** [reply]
In general, only "boring" things like triple-checking all details, and dis/reconnecting all.

**Reply by jabonifacio** [reply]
Thanks anyway
